lib/ui/onboarding_screens/screen3.dart

```diff
@@ -0,0 +1,253 @@
+    List<BottomNavigationBarItem> items = [];
+    for (int i = l; i <= r; i++) {
+      items.add(
+        BottomNavigationBarItem(
+          label: titles[i],
+          icon: BottomBarIcon(asset: icons[i]),
+        ),
+      );
+    }
+    return items;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      resizeToAvoidBottomInset: false,
+      body: Container(
+        width: double.infinity,
+        color: lightColorScheme.surface,
+        padding: const EdgeInsets.only(top: 60, left: 50, right: 50),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Expanded(child: title),
+            Expanded(
+              child: Container(
+                padding: const EdgeInsets.symmetric(vertical: 5),
+                child: desc,
+              ),
+            ),
+            const Spacer(),
+            Expanded(
+              child: Container(
+                padding: const EdgeInsets.only(bottom: 10),
+                alignment: Alignment.bottomCenter,
+                child: const Text(
+                  'Tap on a tool to get more information',
+                  style: TextStyle(
+                    color: Colors.white,
+                    fontSize: 15,
+                  ),
+                  textAlign: TextAlign.start,
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+      bottomNavigationBar: Column(
+        mainAxisSize: MainAxisSize.min,
+        children: [
+          Container(
+            padding: const EdgeInsets.symmetric(vertical: 10),
+            color: lightColorScheme.surface,
+            child: OnboardingPageBottomNavigationBar(
+              onPressedFunctions: [brush, hand, eraser, line],
+              barItems: _getBottomNavigationBarItems(0, 3),
+            ),
+          ),
+          Container(
+            padding: const EdgeInsets.symmetric(vertical: 10),
+            color: lightColorScheme.surface,
+            child: OnboardingPageBottomNavigationBar(
+              onPressedFunctions: [shapes, fill, sprayCan, cursor],
+              barItems: _getBottomNavigationBarItems(4, 7),
+            ),
+          ),
+          Container(
+            padding: const EdgeInsets.symmetric(vertical: 10),
+            color: lightColorScheme.surface,
+            child: OnboardingPageBottomNavigationBar(
+              onPressedFunctions: [text, stamp, transform, importImage],
+              barItems: _getBottomNavigationBarItems(8, 11),
+            ),
+          ),
+          Container(
+            padding: const EdgeInsets.symmetric(vertical: 10),
+            color: lightColorScheme.surface,
+            child: OnboardingPageBottomNavigationBar(
+              onPressedFunctions: [pipette, watercolor, smudge, clipArea],
+              barItems: _getBottomNavigationBarItems(12, 15),
+            ),
```

Comment:
These are all the same, make a class and reuse it please

Reply:
nice catch, made a new class for this